docs/install-manual.md

```diff
@@ -23,6 +23,7 @@ your network.  Each node in your swarm must:
 
 * be accessible by the swarm manager across your network
```

Comment:
@abronan I'd move this to another location.  Also, need to clean up some antecedent issues in the content. I think the `this` in your original paragraph meant the daemon. Check my rewrite please...

```markdown
* be accessible by the swarm manager across your network
* have Docker Engine 1.6.0+ installed
* open a TCP port to listen for the manager
* *do not* install on a VM or from an image created through cloning

Docker generates a unique ID for the Engine that is located in the
`/etc/docker/key.json` file. If a VM is cloned from an instance where a
Docker daemon was previously pre-installed, Swarm will be unable to differentiate
among the remote Docker engines. This is because the cloning process copied the
the identical ID to each image and the ID is no longer unique.

If you forget this restriction and create a node anyway, Swarm displays a single
Docker engine as registered. To workaround this problem, you can generate a new
ID for each node with affected by this issue. To do this stop the daemon on a node,
delete its `/etc/docker/key.json` file, and restart the daemon.

> **Note**: If you use `ssh` to start the Engine `daemon`, make sure the daemon runs in the background. You can do this using `disown` or `nohup` in combination with the `&` (ampersand). Or you could launch the `daemon` through `systemd` or any other Linux-distribution-specific method. Regardless of which launch method you choose, make sure the daemon process does not stop when your session on a node ends.
```